blottertrax/description_provider.py

```diff
@@ -58,3 +58,15 @@ def _get_artist_by_id(self, artist_id: str):
         return self.musicbrainz.get_artist_by_id(
             id=artist_id, includes=['tags', 'ratings', 'annotation', 'url-rels', 'user-tags']
         )['artist']
+    
+    @classmethod
+    def format_network_to_friendly_name(cls, info) -> str:
```

Comment:
Good that you extracted it to a method, made easier to read. Now that you did that could you write a small test for this method, I think it can be static so that you don't need to call the constructor and initialize the Musicbraiz API. After that it's ready to go